generators/app/tasks/createReactApp.js

```diff
@@ -2,7 +2,7 @@ const runCommand = require('./runCommand');
 
 module.exports.installCRA = function installCRA() {
   return runCommand({
-    command: ['npm', ['install', '--global', 'create-react-app']],
+    command: ['npm', ['install', '--global', 'create-react-app@^1.0.0']],
```

Comment:
I'd use @1.5.2 version, the last after version 2

Reply:
It does, when using `^` it takes the last compatible version, which is 1.5.2 right now. I checked it.

Reply:
I'll put ^1.5.2